src/components/Header/DefaultHeader.tsx

```diff
@@ -35,7 +35,7 @@ export function DefaultHeader(): JSX.Element {
         <LoginModal
             open={displayLoginModal}
             onDismiss={(): void => setDisplayLoginModal(false)}
-            loginDescription="Logg inn på for å se en oversikt over dine tavler og for muligheten til å låse nye tavler til din konto."
+            loginCase={'default'}
```

Comment:
Kan kanskje droppe curly brackets for disse, jeg synes i hvert fall det ser penere ut uten.